app/services/date_parser.rb

```diff
@@ -0,0 +1,13 @@
+module DateParser
+  AMERICAN_REGEX = %r{(?<month>\d{1,2})/(?<day>\d{1,2})/(?<year>\d{4})}.freeze
+
```

Comment:
This is a little bit a nitpick: if the regex is not used outside this module, can it be private?

```suggestion
  private_constant :AMERICAN_REGEX
```

Reply:
Good to know! We aren't super worried about visibility in this project, so I'll leave it out for now. No existing usages:

```
identity-idp:> git grep private_constant
>
```